test/web-platform-tests/to-run.yaml

```diff
@@ -392,7 +392,23 @@ custom-element-reaction-queue.html:
   "Mutating a undefined custom element while upgrading a custom element must not enqueue or invoke reactions on the mutated element": [fail, document.write() implementation is not spec compliant]
 element-internals-aria-element-reflection.html:
   "Getting previously-unset ARIA element reflection properties on ElementInternals should return null.": [fail, Unknown]
-form-associated/**: [fail-slow, Not implemented]
+form-associated/ElementInternals-NotSupportedError.html: [fail-slow, Not implemented]
+form-associated/ElementInternals-form.html: [fail-slow, Not implemented]
+# form-associated/ElementInternals-labels.html: [fail-slow, Not implemented]
+# form-associated/ElementInternals-reportValidity-bubble-notref.html: [fail-slow, Not implemented] # cannot be found, even though html file exists?
```

Comment:
These are ref tests, which means they try to render the result in a web browser and then compare iwth the match/mismatch file. We don't support these in the jsdom testing harness because we don't render anything.

You can just omit them from this file.